.circleci/conditional_config.yml

```diff
@@ -336,7 +337,18 @@ jobs:
           name: Run spider
           command: |
             cd src/api
-            bundle exec rake test:spider
+            case $CIRCLE_NODE_INDEX in
+              0)
+                bundle exec rails test test/functional/webui/spider_test.rb -n test_spider_anonymously
+                echo "Test loading the application in production"
```

Comment:
As I already told you @hennevogel, I find this weird to test this in a job (`spider` in this case) which has nothing to do with testing the loading of the application. At the same time, having an extra job only for this is a bit overkill.

Reply:
indeed